test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/HtmlBlockTest/HandlesOpenAngleAtEof.stree.txt

```diff
@@ -1,14 +1,20 @@
-Markup block - Gen<None> - 5 - (0:0,0)
-    Markup span - Gen<Markup> - [] - SpanEditHandler;Accepts:Any - (0:0,0) - Tokens:1
-        SyntaxKind.Unknown;[];
-    Statement block - Gen<None> - 5 - (0:0,0)
-        Transition span - Gen<None> - [@] - SpanEditHandler;Accepts:None - (0:0,0) - Tokens:1
-            SyntaxKind.Transition;[@];
-        MetaCode span - Gen<None> - [{] - SpanEditHandler;Accepts:None - (1:0,1) - Tokens:1
-            SyntaxKind.LeftBrace;[{];
-        Code span - Gen<Stmt> - [LF] - AutoCompleteEditHandler;Accepts:Any,AutoComplete:[}];AtEOL - (2:0,2) - Tokens:1
-            SyntaxKind.NewLine;[LF];
-        Markup block - Gen<None> - 1 - (4:1,0)
-            Tag block - Gen<None> - 1 - (4:1,0)
-                Markup span - Gen<Markup> - [<] - SpanEditHandler;Accepts:Any - (4:1,0) - Tokens:1
-                    SyntaxKind.OpenAngle;[<];
+RazorDocument - [0..5) - FullWidth: 5 - [@{LF<]
```

Comment:
Interesting, so only the first node in every file spits out the corresponding content? Hmmm

Reply:
Yes. It spits out the entire content of the tree aka the input

Reply:
Might be valuable to spit out content of each tag block